WeScan/Common/EditScanCornerView.swift

```diff
@@ -13,13 +13,54 @@ final class EditScanCornerView: UIView {
     
     let position: CornerPosition
     
+    /// The image to display when the corner view is highlighted.
+    private var image: UIImage?
+    private(set) var isHighlighted = false
+    
+    lazy private var circleLayer: CAShapeLayer = {
+        let layer = CAShapeLayer()
+        layer.fillColor = UIColor.clear.cgColor
+        layer.strokeColor = UIColor.white.cgColor
+        layer.lineWidth = 1.0
+        return layer
+    }()
+    
     init(frame: CGRect, position: CornerPosition) {
         self.position = position
         super.init(frame: frame)
+        backgroundColor = UIColor.clear
+        clipsToBounds = true
+        layer.addSublayer(circleLayer)
     }
     
     required init?(coder aDecoder: NSCoder) {
         fatalError("init(coder:) has not been implemented")
     }
     
+    override func layoutSubviews() {
+        layer.cornerRadius = bounds.width / 2.0
```

Comment:
How does this work performance wise? And shouldn't we call super?

Reply:
Not sure performance wise either. I haven't had a chance to try it in another way.
Do you have a suggestion to improve performance? 
We should definitely call `super`. Good catch!

Reply:
This is often the way to do it, but I'm not sure if that works for this case: http://skladek.github.io/performance/2016/03/16/rounded-corners.html

Reply:
Yeah, the problem here is that I need to draw the image with rounded corners.